### PR TITLE
Rayjames add page

### PR DESCRIPTION
**Pull request by rayjames**
Well crap.  I created an earlier pull request but closed it when it came up with errors.  I don't think I should have done that.  Oh well, I'm learning though.  This is the new pull request with the correct indentation in the file.  Let's hope it's error free.

### CODE REVIEW

**Comment by rayjames** [comment]
Yay!  It's ready for the merge.

File: _pages/rayjames.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: rayjames
+pageUrl: rayjames.github.io
+timestamp: TODAYS-DATE-2016-12-06
```
Comment (hectorsector) [inline _pages/rayjames.yaml]: @rayjames just take out the `TODAYS-DATE-` portion

**Comment by rayjames** [comment]
Ok, all good.  Thanks for catching that date section.  Woot!

**Comment by rayjames** [comment]
Hi @hectorsector, my merge button is grayed out and next to it says "You're" not authorized to merge this pull request.  Please advise. Thanks.

**Comment by hectorsector** [comment]
Sorry about that @rayjames. Try now!

**Pull request merged**